.travis.yml

```diff
@@ -17,6 +17,8 @@ cache:
 env:
   global:
     - VERILATOR_ROOT=$TRAVIS_BUILD_DIR
+    # Hack around debug mode issues on FreeBSD
+    - ENV_HACK=$([ ! -e $TRAVIS_BUILD_DIR/ci/travis-env-hack.bash ] || $TRAVIS_BUILD_DIR/ci/travis-env-hack.bash)
```

Comment:
Does something use the ENV_HACK value?

I'd prefer this and the filename get renamed to indicate what it does, e.g, travis-env-shebang or something.

Reply:
Nothing uses the value, the only purpose is to execute the script early during the build in order to monkey around with the system so Travis's own scripts don't crash... This is the best (albeit gross) injection method I could find. Hopefully Travis will get fixed soon so this will go. Actually the only thing that needs this now is Travis debug mode so you can get an SSH terminal. Whit that in mind what I will do is remove this entirely in a separate non-squashed patch (so we can still get it back easily if we need debug SSH on Travis FreeBSD before it is actually fixed).